src/test/resources/catalogs/default/EXCLUDE_T_NESTED_LIST.ion

```diff
@@ -15,15 +15,15 @@
               fields: [
                 {
                   name: "field_x",
-                  type: ["int32", "null"],
+                  type: "int32",
```

Comment:
self-review: since PLK-0.14.x defined `null` as a type, some prior table definitions had used "null" to denote a type is nullable. In PLK-1.x's type system, all types are nullable so some test table schemas are modified. Some modified tables would have the same schemas as the original "non-null" tables (e.g. T_INT32.ion and T_INT32_NULL.ion). I've included the nullable tables for posterity with the suffix `...ion_ignore`.

Reply:
I'll take another look at this

Reply:
Deleted the `.ion_ignore` tables that
- included the `null` type
- duplicated the schemas of tables that did not include the `null` type

[`7fa1825` (#74)](https://github.com/partiql/partiql-scribe/pull/74/commits/7fa1825bd57e4c95c591ca2a65d5ac1c870f7238)